src/Microsoft.PowerApps.TestEngine/TestInfra/PlaywrightTestInfraFunctions.cs

```diff
@@ -105,6 +105,14 @@ public async Task SetupAsync(IUserManager userManager)
                 {
                     launchOptions.Channel = browserConfig.Channel;
                 }
+                else
+                {
+                    if (browserConfig.Browser?.ToLower() == "chromium")
+                    {
+                        // Default to msedge browser
+                        launchOptions.Channel = "msedge";
+                    }
```

Comment:
can we reset the channel to input value from the user if it is specifically requested in the yaml